test/test_echo_server.rs

```diff
@@ -87,7 +87,9 @@ impl EchoConn {
             }
             Err(e) => {
                 debug!("not implemented; client err={:?}", e);
-                if let Some(x) = self.interests.as_mut() {
+                if self.interests == Some(Interests::readable()) {
```

Comment:
Would this be too much for just remove `Interests::readable()`?

Reply:
Are you asking if you can remove the `if` condition here?

Reply:
I don't understand why this bit needs to change.

Reply:
Ah, the answer is in [this](https://github.com/tokio-rs/mio/issues/934#issuecomment-492096665) comment.

Reply:
LGTM, just ignore it.